game-core/src/test/resources/Total_World_War_Dec1941.xml

```diff
@@ -4,6 +4,66 @@
   <info name="Total World War: December 1941" version="2.7.7.2"/>
   <triplea minimumVersion="1.8"/>
   <diceSides value="12"/>
+  <variableList>
+    <variable name="AntiTankTargets">
+      <element name="germanMech.Infantry"/>
+      <element name="germanTank"/>
+      <element name="germanHeavyTank"/>
+      <element name="russianMech.Infantry"/>
+      <element name="russianTank"/>
+      <element name="russianHeavyTank"/>
+      <element name="americanMech.Infantry"/>
+      <element name="americanTank"/>
+      <element name="americanHeavyTank"/>
+      <element name="italianMech.Infantry"/>
+      <element name="italianTank"/>
+      <element name="italianHeavyTank"/>
+      <element name="japaneseMech.Infantry"/>
+      <element name="japaneseTank"/>
+      <element name="japaneseHeavyTank"/>
+      <element name="britishMech.Infantry"/>
+      <element name="britishTank"/>
+      <element name="britishHeavyTank"/>
+      <element name="chineseMech.Infantry"/>
+      <element name="chineseTank"/>
+      <element name="spanishTank"/>
+      <element name="brazilianTank"/>
+      <element name="turkishTank"/>
+      <element name="swedishTank"/>
+    </variable>
+    <variable name="russianAntiAirTargets">
+      <element name="russianFighter"/>
+      <element name="russianTacticalBomber"/>
+    </variable>
+    <variable name="americanAntiAirTargets">
+      <element name="americanFighter"/>
+      <element name="americanTacticalBomber"/>
+    </variable>
+    <variable name="AntiAirTargets">
+      <element name="russianAntiAirTargets"/>
+      <element name="americanAntiAirTargets"/>
+    </variable>
+    <variable name="Infantry">
+      <element name="germanInfantry"/>
+      <element name="russianInfantry"/>
+    </variable>
+    <variable name="Players">
+      <element name="Germany"/>
+      <element name=""/>
```

Comment:
Quick question @ron-murhammer, are empty values intentional and allowed?

Reply:
Yes, empty values are allowed and essentially means that element will be skipped if used in the foreach.

Reply:
What's the use-case for that? I'm wondering why not just leave the element off.

Reply:
You'll often have very similar attachments where some of them have say a unit option and some don't. The reason you can't just leave it off is ordering as the one that doesn't have it might not always be the last element. So imagine you had 3 more players defined after that blank and you want the second attachment generated not to have the option not the last one to not have it. Here is an example: https://forums.triplea-game.org/topic/560/categories-and-or-tags/28

Reply:
I think naming them "sequence" instead of "variable" may help getting this concept.